instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/InetSocketAddressNetClientAttributesGetter.java

```diff
@@ -21,32 +21,26 @@
     implements NetClientAttributesGetter<REQUEST, RESPONSE> {
 
   @Nullable
-  public abstract InetSocketAddress getAddress(REQUEST request, @Nullable RESPONSE response);
+  public abstract InetSocketAddress getPeerAddress(REQUEST request, @Nullable RESPONSE response);
```

Comment:
can this method be `protected`?

Reply:
what do you think of renaming it to `getPeerSocketAddress`? I kept getting this method confused when reading later on in the PR, whether this was used to calculate net.peer.* or net.sock.peer.*